source/fundamentals/crud/read-operations/retrieve.txt

```diff
@@ -63,6 +63,49 @@ filter as a ``SingleResult`` type.
 To learn how to access data in a ``SingleResult`` see the :ref:`BSON
 <bson-unmarshalling>` guide.
 
+Modify Behavior
+~~~~~~~~~~~~~~~
+
+The ``Find()`` and ``FindOne()`` functions optionally take a
+``FindOptions`` and ``FindOneOptions`` type, which represents options
+you can use to modify their behavior. If you don't specify any options,
+the driver uses the default values for each option.
+
+You can configure the commonly used options in both types with the
+following functions:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Function
+     - Description
+
+   * - ``SetCollation()`` 
+     - | The type of language collation to use when sorting results.  
+       | Default: ``nil``
+
+   * - ``SetLimit()`` 
+     - | The maximum number of documents to return. 
+       | Default: ``0`` 
+
+       .. note::
+
+          This option is not available for ``FindOneOptions``. The
+          ``FindOne()`` funciton internally uses ``SetLimit(-1)``.
+
+   * - ``SetProjection()`` 
+     - | The fields to include in the returned documents. 
+       | Default: ``nil``
+
+   * - ``SetSkip()`` 
+     - | The number of documents to skip.
+       | Default: ``0``
+
+   * - ``SetSort()`` 
+     - | The order in which to return matching documents
+       | Default: to not specify a sort
```

Comment:
"None"?
```suggestion
       | Default: none
```

Reply:
I confirmed with an engineer that it's more accurate to say not specifying a value for the field. Although `none` is somewhat synonymous, I think it would be more accurate to leave it as is.

Reply:
What probably irks me is that you have a verb here when all of the other default values are nouns.